FrontEnd/src/components/SignUp/components/signup/signup-form/RulesModal.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './RulesModal.module.css';
+import rulesText from './Rules_text';
+
+const RulesModal = ({ closeModal }) => {
+  return (
+    <div className={styles.modal}>
```

Comment:
Do we have `modal` selector in the `./RulesModal.module.css`?

Reply:
done